Microsoft.Toolkit.Uwp.Notifications/Microsoft.Toolkit.Uwp.Notifications.nuspec

```diff
@@ -6,9 +6,17 @@
     <title>Windows Community Toolkit Notifications</title>
     <authors>Microsoft.Toolkit,dotnetfoundation</authors>
     <requireLicenseAcceptance>true</requireLicenseAcceptance>
-    <licenseUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/license.md</licenseUrl>
-    <projectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</projectUrl>
-    <iconUrl>https://raw.githubusercontent.com/windows-toolkit/WindowsCommunityToolkit/master/build/nuget.png</iconUrl>
+  <PackageProjectUrl>https://github.com/windows-toolkit/WindowsCommunityToolkit</PackageProjectUrl>
+<package>
+  <metadata>
+    <icon>images\nuget.png</icon>
+    <license type="file">License.md</license>
+  </metadata>
+  <files>
+    <file src="WindowsCommunityToolkit\build\nuget.png" target="images\" />
+    <file src="WindowsCommunityToolkit\License.md" target="" />
```

Comment:
These two files can go in the files block below. We're in the 'Microsoft.Toolkit.Uwp.Notifications' directory here, so the src path should be ..\build\nuget.png and ..\license.md I believe. Like:

`<file src="..\build\nuget.png" target="images\nuget.png" />`